moveit_ros/planning_interface/move_group_interface/src/move_group.cpp

```diff
@@ -155,44 +193,35 @@ class MoveGroup::MoveGroupImpl
   }
 
   template<typename T>
-  void waitForAction(const T &action, const ros::Duration &wait_for_server, const std::string &name)
+  void waitForAction(const T &action, const std::string &name, const ros::WallTime &timeout, double allotted_time)
```

Comment:
why did you move the ``name`` var to the middle of the function header instead of at the back? seems like this is an overly-intrusive re-ordering